polylogue/cli/completion_engine.py

```
@@ -0,0 +1,282 @@
+            return [Completion("__PATH__")]
+        if subcmd == "stats" and current_word.startswith("--"):
+            return self._option_completions("inspect stats")
+        return []
+
+    def _complete_watch(self, args: Sequence[str], current_index: int) -> List[Completion]:
+        if current_index == 1:
+            return [Completion(name) for name in LOCAL_SYNC_PROVIDER_NAMES]
+        prev = args[current_index - 1] if current_index > 0 else ""
+        if prev in {"--base-dir", "--out"}:
+            return [Completion("__PATH__")]
+        if current_word.startswith("--"):
```

Ruff (0.15.9)

[error] Undefined name `current_word`

(F821)